nucleus/__init__.py

```diff
@@ -1249,6 +1250,27 @@ def create_image_index(self, dataset_id: str):
             requests_command=requests.post,
         )
 
+    def add_taxonomy(
```

Comment:
define this on the dataset class, no more additions to this gigantic file please :)